frontend/src/libs/packages/store/middlewares/middlewares.ts

```diff
@@ -1,16 +1,22 @@
 import { type AnyAction, type Middleware } from '@reduxjs/toolkit';
 import { isRejected } from '@reduxjs/toolkit';
+import { ExceptionMessage } from 'shared/build/index.js';
```

Comment:
I'm not sure that import from `shared` folder is allowed to be here, only via reexports